Frameworks/include/CoreGraphics/D2DWrapper.h

```diff
@@ -22,6 +22,7 @@
 #import "Wincodec.h"
```

Comment:
no?

---
In reply to: [88943733](https://github.com/Microsoft/WinObjC/pull/1384#discussion_r88943733) [](ancestors = 88943733)